src/zos_files/zowe/zos_files_for_zowe_sdk/file_system.py

```diff
@@ -18,42 +20,42 @@
 
 class FileSystems(SdkApi):
     """
-    Class used to represent the base z/OSMF FileSystems API
-    which includes all operations related to file systems.
+    Class used to represent the base z/OSMF FileSystems API.
 
-    ...
+    It includes all operations related to file systems.
 
-    Attributes
+    Parameters
     ----------
-    connection
-        connection object
+    connection : dict
+        A profile for connection in dict (json) format
     """
 
-    def __init__(self, connection):
-        """
-        Construct a FileSystems object.
-
-        Parameters
-        ----------
-        connection
-            The z/OSMF connection object (generated by the ZoweSDK object)
-
-        Also update header to accept gzip encoded responses
-        """
+    def __init__(self, connection: dict):
         super().__init__(connection, "/zosmf/restfiles/", logger_name=__name__)
         self._default_headers["Accept-Encoding"] = "gzip"
 
-    def create(self, file_system_name, options={}):
+    def create(self, file_system_name: str, options: dict = {}) -> dict:
         """
         Create a z/OS UNIX zFS Filesystem.
 
-        Parameter
-        ---------
-        file_system_name: str - the name for the file system
+        Parameters
+        ----------
+        file_system_name: str
+            the name for the file system
+        options: dict
+            specifies file system attributes
```

Comment:
I think we should use sentence case to make the descriptions consistent
```suggestion
        file_system_name: str
            Name of the file system
        options: dict
            Specifies file system attributes
```